Microsoft.HealthVault.Fhir/Transformers/PersonItemToFhir.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
The other files that GRH has added had a copyright attributed to them.

Reply:
Changed to Get Real Health.